### PR TITLE
create recipe for r-leafsync using skeleton-helper

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by agcopenhaver** [comment]
@oblute @Tata17 do you agree to be maintainers?

**Comment by Tata17** [comment]
Yes

**Comment by agcopenhaver** [comment]
@conda-forge-admin, please lint

File: recipes/r-leafsync/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+test:
+  commands:
+    - $R -e "library('leafsync')"           # [not win]
+    - "\"%R%\" -e \"library('leafsync')\""  # [win]
+
+about:
+  home: https://github.com/r-spatial/leafsync
+  license: MIT
+  license_file: LICENSE
```
Comment (cbrueffer) [inline recipes/r-leafsync/meta.yaml]: Please change this to
```license_file:
       - LICENSE
       - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
```

Comment (agcopenhaver) [reply recipes/r-leafsync/meta.yaml]: Thanks

**Comment by cbrueffer** [comment]
Thanks @agcopenhaver !

**Pull request merged**